src/codeu/chat/server/Server.java

```diff
@@ -143,7 +139,30 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
       }
     });
 
-    // Get Conversations By Id - A client wants to get a subset of the converations from
+    // Get Server Request - A client whats to get all the request to the server from the back end.
+    this.commands.put(NetworkCode.SERVER_INFO_REQUEST, new Command(){
+        @Override
+        public void onMessage(InputStream in, OutputStream out) throws IOException{
+            Serializers.INTEGER.write(out, NetworkCode.SERVER_INFO_RESPONSE);
+
+
+            try{
+              serverInfo = new ServerInfo();
+            }catch (Exception ex){
```

Comment:
Over-catching is not healthy. Keep your exception catching to declared methods only or to methods that you really expect. Avoid catch generic exceptions such as Exception or Throwable or RuntimeException.